src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -123,13 +121,13 @@ class RtpSenderImpl(
         private const val INCOMING_DURATION_MS = "incoming_duration_ms"
         private const val SENT_BYTES = "sent_bytes"
         private const val SENT_DURATION_MS = "sent_duration_ms"
-        private const val QUEUE_NUM_READS = "queue_num_reads"
-        private const val QUEUE_READ_DURATION_S = "queue_read_duration_s"
     }
 
     init {
         logger.cdebug { "Sender $id using executor ${executor.hashCode()}" }
 
+        incomingPacketQueue.setErrorHandler(queueErrorCounter)
```

Comment:
could we just do `incomingPacketQueue.setErrorHandler(CountingErrorHandler())`?

Reply:
No, this one is static (in the companion object)